MyApp/app/controllers/application_controller.rb

```diff
@@ -13,6 +13,6 @@ class ApplicationController < ActionController::Base
 
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
-    devise_parameter_sanitizer.permit(:account_update, keys: [:name])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:name, :image, :full_name, :role, :bio])
```

Comment:
You should permit this params is user_controller, and also, do update for user in users_controler, not registraton, 
same for show action, which I dont see but description says: View user's profile

Reply:
I didn't understand this quite right. I am using devise gem for authentication and in the tutorial, they advised to place all methods regarding user (new / edit..) in registrations_controller, so I followed the comments.

Reply:
ok, then move list of permitter params in scope of that controller

Reply:
Here it is: https://github.com/heartcombo/devise